README.md

```diff
@@ -26,22 +26,21 @@ Before you start, make sure that you have setup the supported version of Python
 
 | Input                  | Required                            | Description        |
 | ---------------------- | ----------------------------------- | ------------------ |
+| `token`                | ✅ Yes                              | GitHub token |
 | `source_dir`           | ✅ Yes                              | Input directory    |
 | `destination_dir`      | ❌ No (default to `source_dir`)     | Output directory   |
 | `signing_key`          | ❌ No                               | Signing key        |
 | `commit_message`       | ❌ No (default to `JSON Cleanup`)   | Commit message     |
-| `secrets.GITHUB_TOKEN` | ❌ No (default to GitHub's default) | GitHub's PAT token |
 
 ```yml
   - name: Nested JSON Postprocessor
     uses: validcube/crowdin-nested-json-postprocessor@v0.1
     with:
+      token: ${{ secrets.GITHUB_TOKEN }}                        # required, your push token
```

Comment:
#### 👋 Suggested change:

Can the `token` value in the example use `${{ github.token }}` instead of `${{ secrets.GITHUB_TOKEN }}`?

`${{ secrets.GITHUB_TOKEN }}` required the user to set the token as a secret variable, which might prompt us to write another section or paragraph explaining how to create PAT, while `${{ github.token }}` is already set by the runner.